### PR TITLE
issue #36 - fixing problem with processing wsdl from url

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "displayName": "wsdl2rest",
     "description": "A VS Code extension supporting wsdl2rest via Camel Rest DSL",
     "license": "Apache-2.0",
-    "version": "0.0.4",
+    "version": "0.0.5",
```
Comment (apupier) [inline package.json]: I'm expecting the package-lock.json to be updated too then.
Apart from that, I think it is a good practice to update the version in a separate PR as soon as the previous release has been published.

File: src/extension.ts [hunk 2]
```diff
@@ -187,7 +187,7 @@ function callWsdl2Rest(wsdl2restExecutablePath: string): Promise<boolean> {
 
 			let outPath: string = path.join(storagePath, outputDirectory);
 			
-			if (!(wsdlFileUri.startsWith('http:') || !wsdlFileUri.startsWith('https:'))) {
+			if (!((wsdlFileUri.startsWith('http:') || wsdlFileUri.startsWith('https:')))) {
```
Comment (apupier) [inline src/extension.ts]: the added parentheses are useless. The issue was the !

(SonarLint is reporting the useless parentheses)

**Comment by bfitzpat** [comment]
> 
> 
> What do you think to add a test using an https URL? Not sure how complicated it is to provide the service used in test through both http and https.

That's a good idea, but I'm not sure how to handle it honestly. The simple service is just http. Looks like this has some ideas on adding https support - https://www.alexkras.com/simple-https-test-server-using-node-js/.

Not sure that's a good use of time at this point, but I can add another issue to look into it down the line.

**Comment by bfitzpat** [comment]
Spun off https://github.com/camel-tooling/vscode-wsdl2rest/issues/39

**Comment by bfitzpat** [comment]
Fixed sonar-reported issue and updated changelog

File: package.json (hunk 1 above)
Comment (bfitzpat) [reply package.json]: Updated package-lock

**Pull request merged**